azure_functions_worker/loader.py

```diff
@@ -225,7 +227,9 @@ def index_function_app(function_path: str):
                     f"level function app instances are defined.")
 
     if not app:
+        script_file_name = get_app_setting(setting=SCRIPT_FILE_NAME,
+                                           default=SCRIPT_FILE_NAME_DEFAULT)
         raise ValueError("Could not find top level function app instances in "
-                         f"{SCRIPT_FILE_NAME}.")
+                         f"{script_file_name}.")
```

Comment:
We should add an example or an aka.ms link here on how they should program this app. Check in the logs how many apps are hitting this error today?

Reply:
![image](https://github.com/Azure/azure-functions-python-worker/assets/80794152/cb3a7c87-78cd-43d6-aa99-86ccc51161d3)

This is the worker log. Can customers see this in app insights?